CMakeLists.txt

```diff
@@ -165,7 +165,26 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 endif()
 
 # Set the installation path for shared libraries
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+if (APPLE)
+    set(RPATH_RELATIVE_SYMBOL "@loader_path")
+else ()
+    # this should just be ignored on windows...
+    set(RPATH_RELATIVE_SYMBOL "$ORIGIN")
+endif()
+
+# We want the RPATHS to be relative, so the libraries can be relocated; however,
+# we also include the absolute RPATH as a last option because:
+# - the github tests copy the python libs to the python site packages directory;
+#   from that location, the python libs can't find the MaterialX libs at their
+#   "standard" relative location
+# - for maximum backward compatibility - in case other end users are relying on
+#   the absolute paths (like the github tests are)
+# For linking to libraries in the same dir - ie, things in "MATX/lib" to each other
+set(MATERIALX_SAME_DIR_RPATH "${RPATH_RELATIVE_SYMBOL};${CMAKE_INSTALL_PREFIX}/${MATERIALX_INSTALL_LIB_PATH}")
```

Comment:
Is this line correct?  My sense of symmetry leads me to expect that the first path should be `${RPATH_RELATIVE_SYMBOL}/${MATERIALX_INSTALL_LIB_PATH}`, so that it matches the pattern in the other two `RPATH` variables.  But it's very possible that my sense of symmetry is misguided in this case, and that the current logic is correct.

Reply:
Yes, I believe this is correct - the name of this var is "MATERIALX_SAME_DIR_RPATH" - so it's intended to be used on targets thar should search for their dependencies in the same directory as they are located - ie, it will generally be used on libraries that are in the lib dir, and need to find other libraries sitting beside them in the same lib dir.

It's also tacked into the end of the other MATERIALX_..._RPATH dirs, so will also check beside themselves for binaries in the same dir.